cherryontech/soft-skill-generator
# Header component, project colors added to tailwind.config file

## Description:

Header component is created with logo.
Hex codes added as available colors to tailwind.config file
Logo (in proper colors) added to assets folder
*note: I have commented out the 'hello world test deploy' h1 in app.tsx folder*

## Testing / Acceptance Criteria

- Pull down branch
- check tailwind.config.js file for presence of the following:
extend: {
      colors: {
          50: '#eef26d', //yellow
          100: '#b796d9', //light purple
          150: '#4e2973', //dark purple
          200: '#f2f2f2', //white
          250: '#0d0d0d', //black
        }
      },
- verify logo.jpeg file in assets folder
- launch app and verify presence of header

## Checklist:

- [ ✅ ]I have tested the changes locally.
- [✅ ] I have **requested review** from the other 2 developers on the project
